Header/componentsHeader.css

```diff
@@ -0,0 +1,43 @@
+ /* header */
+
+ 
+ .header-nav{
+    background: black;
+    height: 80px;
+    width: 100%;
+  }
+  
+.header-nav ul{
+    float: right;
```

Comment:
While `float:`  works, try using `flexbox as much as you can`. You will have less trouble making it responsive that way. Flexbox is much more modular and flexible (pun intended :P )